evaluate/evaluate.go

```diff
@@ -249,7 +249,8 @@ func Evaluate(ctx *Context) (assessments *report.AssessmentStore, totalScore uin
 		}
 	}
 	if isOnlyPlainRepositories {
-		totalScore = uint64(len(ctx.Languages)) * uint64(ctx.Runs)
+		// For each task, the `symflower fix` is also called so we multiply the total score by 2.
```

Comment:
For every write-test task in the plain repository, each model is also executed with `symflower fix` which results in double the total results.